src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,30 @@
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    name,
+    email,
+    body,
+  },
+}) => (
+  <>
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{name}</strong>
+
+      {' by '}
```

Comment:
```suggestion
      by
```
you may use a plain text in the jsx